server/views/challenges/showStep.jade

```diff
@@ -14,7 +14,7 @@ block content
                     .spacer
                     .challenge-button-block
                         if step[3]
-                            a.btn.btn-block.btn-primary.challenge-step-btn-action.btn-lg(id='#{index}' href='#{step[3]}' target='_blank') Open link in new tab (this unlocks the next step)
+                            a.btn.btn-block.btn-primary.challenge-step-btn-action.btn-lg(id='#{index}' href='#{step[3]}' target='_blank') Confirm
```

Comment:
As mentioned by @BerkeleyTrue in his [comment](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/8261#issuecomment-214136164), this has to be a conditional check.

The current change will affect all getting started challenges.